velox/docs/functions/spark/aggregate.rst

```diff
@@ -7,6 +7,14 @@ Aggregate functions operate on a set of values to compute a single result.
 General Aggregate Functions
 ---------------------------
 
+.. spark:function:: avg(x) -> double|decimal
+
+    Returns the average (arithmetic mean) of all input values.
+    When x is of type DECIMAL, the result type is DECIMAL.
+    For all other input types, the result type is DOUBLE.
+    When all inputs are nulls, the result of partial avg is zero for sum and count,
```

Comment:
Perhaps, we should start including intermediate type in these docs.

`Intermediate results are (sum, count) pairs represented as row(double, bigint).`

Reply:
Added.